workspace/exercise_metadata.go

```diff
@@ -10,14 +10,14 @@ import (
 	"time"
 )
 
-const solutionFilename = "solution.json"
-const legacySolutionFilename = ".solution.json"
+const metadataFilename = "metadata.json"
```

Comment:
We should probably add a note for this in the release notes as it could cause confusion for any users who version control their workspace files.